jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/JibPluginConfiguration.java

```diff
@@ -161,17 +163,61 @@ private static PlatformParameters of(String osArchitecture) {
     }
   }
 
+  /** Configuration for {@code [from|to].credHelper} parameter. */
+  public static class CredHelperParameters implements CredHelperConfiguration {
+    @Nullable @Parameter private String helper;
+    @Parameter private Map<String, String> environment;
```

Comment:
ditto
```suggestion
    @Parameter private Map<String, String> environment = new HashMap<>();
```
and remove the constructor.